[
  {
    "class": "django.core.exceptions.ValidationError",
    "message": "['Anonymous reviews must include a name and an email']",
    "stack": [
      "src/oscar/apps/catalogue/reviews/abstract_models.py:98"
    ]
  }
]